app/lib/communicator.js

```diff
@@ -270,7 +270,7 @@ const getUpdatedRequestHeaders = (
     headers = {},
     currentRequest = getCurrentRequest()
 ) => {
-    const clientUserAgent = currentRequest?.headers['user-agent'];
+    const clientUserAgent = currentRequest?headers['user-agent']:null;
```

Comment:
Not sure if this is correct. Are you perhaps running an old NodeJS version that doesn't support [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)?

In that case make sure you delete `node_modules` entirely when you update to NodeJS 14.

Reply:
Yes, that might be the case. I'm still running node 12. 
Code reverted for communicator.js, thanks @MartijnR